randomizers/hints.py

```diff
@@ -12,22 +12,29 @@
 from asm import patcher
 
 
-class HintType(Enum):
-  PATH = 0
-  BARREN = 1
-  ITEM = 2
-  LOCATION = 3
-  FIXED_LOCATION = 4
+class HintType(Flag):
+  PATH = 1
+  BARREN = 2
+  ITEM = 4
+  LOCATION = 8
+  FIXED_LOCATION = 16
+
+
+class ItemImportance(Flag):
+  REQUIRED = 1
+  POSSIBLY_REQUIRED = 2
+  NOT_REQUIRED = 4
```

Comment:
Why is Flag being used instead of Enum for these two? You don't seem to be combining multiple together. And you also don't seem to be using bitwise operators or `in`.

Reply:
Oh, good catch. I had used the `Flag` property while working on the feature, but didn't use it in the end. I can revert those changes.